cdap-ui/app/features/services/controllers/tabs/runs/tabs/status/make-request-ctrl.js

```diff
@@ -108,8 +110,12 @@ angular.module(PKG.name + '.feature.services')
       }
 
       dataSrc.request(requestObj)
-        .then(function(res) {
+        .then(function success(res) {
           vm.response = res;
+          vm.loading = false;
+        }, function error(err) {
+          vm.response = 'Some error has occured: ' + err;
```

Comment:
@elfenheart Its just showing [Object object] in the browser. I think the backend is returning only string as error (be it stack trace too). We could probably stick to just string.